CPS-????/README.md

```diff
@@ -0,0 +1,93 @@
+---
+CPS: ?
+Title: Coin Selection Including Native Tokens
+Status: Open
+Category: Wallets
+Authors:
+  - Hinson Wong <wongkahinhinson@gmail.com>
+  - Tsz Wai Wu <wu.tszwai@outlook.com>
+Proposed Solutions:
+  - []
```

Comment:
```suggestion
Proposed Solutions: []
```
Proper definition of an empty list according to CIP-0001 ([and YAML](https://stackoverflow.com/questions/5110313/how-do-i-create-an-empty-array-in-yaml)).